applications/nrf_desktop/doc/dfu.rst

```diff
@@ -212,35 +213,37 @@ sync
 
    The update tool can fetch the ``sync`` option before starting the update process to see at which offset the update is to be restarted.
 
-   Fetching the ``sync`` option also triggers moving the image data from the RAM buffer to flash.
+   Fetching the ``sync`` option also triggers moving the image data from the RAM buffer to non-volatile memory.
 
 .. _dfu_bootloader_var:
 
 module_variant
    Perform the fetch operation on this option to get the information about the bootloader variant that is being used on the device.
 
 
-Writing data to flash
-=====================
+Writing data to non-volatile memory
+===================================
```

Comment:
Shouldn't the dfu_lock.rst also be updated (line no: 24). We have mention of flash memory.

Reply:
It will be updated in a separate PR.